docs/concepts/gas.md

```diff
@@ -152,9 +152,7 @@ Function calls are more complex, and you can attach an explicit amount of gas to
 
     near call myContract.testnet myFunction "{ \"arg1\": \"val1\" }" --gas=300000000000000
 
-And in [`near-api-js`](https://github.com/near/near-api-js), given a `contract` that you've instantiated with `new Contract`, you can attach an explicit gas amount when calling a change method on this contract with:
-
-    contract.myFunction({ arg1: 'val1' }, '300000000000000')
+And in [`near-api-js`](https://github.com/near/near-api-js), you can also specify an explicit amount of gas units to attach when calling a change method; see [example here](https://github.com/near-examples/guest-book/blob/ceb2a39e53351b4ffc21d01987e2b0b21d633fa7/src/App.js#L29).
```

Comment:
Curious why we want the user to click away for a line of code example?

Reply:
Because it's actually many lines of code, plus an entire `src/config.js` file. @janedegtiareva and I discussed and thought maybe it was better to link to an example rather than bulk this page with all that boilerplate. 

Here's how we usually bootstrap a `contract` object so you can finally call your change method on it:

```js
import getConfig from './config.js'
import { connect, Contract, keyStores, WalletConnection } as nearAPI from 'near-api-js'

const nearConfig = getConfig(process.env.NODE_ENV || 'testnet')

async function initContract() {
  window.near = await connect({
    deps: {
      keyStore: new keyStores.BrowserLocalStorageKeyStore()
    },
    ...nearConfig
  })

  window.walletConnection = new WalletConnection(near)

  window.contract = await new Contract(walletConnection.account(), nearConfig.contractName, {
    changeMethods: ['myFunction'],
  })
}

window.nearInitPromise = initContract()

// then later, when some event happens
contract.addMessage({ arg1: "val1" }, '300000000000000')
```

I think all of this boilerplate can and should go away, but until then, I don't see a great way to simplify it to a point that it fits into this document without being a bit distracting. 

What do you think?